Add a fedtidy prow config for release 1.7.

madhusudancs opened the pull request:
Note that this will only work for release 1.7 for now. The code will soon be updated to handle multiple projects.

/assign @spxtr @csbell

krzyzacy commented on prow/config.yaml:
```diff
@@ -15826,6 +15826,31 @@ periodics:
+      env:
+      - name: GOOGLE_APPLICATION_CREDENTIALS
+        value: /etc/service-account/service-account.json
+      image: gcr.io/k8s-testimages/fedtidy:0.1
```
can you commit the image to /images as well?

madhusudancs replied:
I can for sure. Tricky part is getting the binary. What's the recommendation?

krzyzacy replied:
^^

spxtr commented on prow/config.yaml:
```diff
@@ -15826,6 +15826,31 @@ periodics:
+      - --project=k8s-jkns-e2e-gce-f8n-1-7
+      - --dns-zone=release1-7-test-f8n-k8s-io
+      command:
+      - /fedtidy
```
You can bake this into the image.

madhusudancs replied:
Done.

madhusudancs commented:
@spxtr @krzyzacy PTAL. I updated the PR.

@spxtr I added the configmap as we discussed. I did not add the non-vendored code. I am not entirely sure adding that would help. However, I will be very happy to move the code along with the vendored libraries here.

krzyzacy commented:
(add an entry to testgrid to make travis happy)

madhusudancs commented:
@krzyzacy done.

krzyzacy commented on experiment/fedtidy/configmap.yaml:
```diff
@@ -0,0 +1,21 @@
+        "project": "k8s-jkns-pr-bldr-e2e-gce-fdrtn",
+        "dnsZone": "pr-bldr-test-f8n-k8s-io"
+      }
+    ]
```
nitpick: new line

krzyzacy commented:
Maybe add some comment describe where the binary comes from or something? As now I have no idea what is this fedtidy thing..

krzyzacy commented on testgrid/config/config.yaml:
```diff
@@ -562,6 +562,8 @@ test_groups:
 - name: maintenance-pull-janitor
   gcs_prefix: kubernetes-jenkins/logs/maintenance-pull-janitor
+- name: maintenance-maintenance-fedtidy
+  gcs_prefix: kubernetes-jenkins/logs/maintenance-maintenance-fedtidy
```
one less maintenance :-)

fejta commented:
/retest
ref https://github.com/kubernetes/test-infra/issues/4568

krzyzacy commented:
is this still needed? or it's running from somewhere else?

madhusudancs commented:
It's running on my computer now. Let's hold off on this a bit.
